libs/wingsdk/src/target-tf-aws/schedule.ontick.inflight.ts

```diff
@@ -0,0 +1,16 @@
+import type {
+  IFunctionHandlerClient,
+  IScheduleOnTickHandlerClient,
+} from "../cloud";
+
+export class ScheduleOnTickHandlerClient
+  implements IScheduleOnTickHandlerClient
+{
+  private readonly handler: IFunctionHandlerClient;
+  constructor({ handler }: { handler: IFunctionHandlerClient }) {
+    this.handler = handler;
+  }
+  public async handle(event: any) {
+    await this.handler.handle(event.detail);
```

Comment:
How do we know event has a `detail` field? We should annotate `event` with an interface like `event: AwsScheduleEvent` for documentation

Reply:
I followed the way the topic was implemented

How about importing @types/aws-lambda into the project and using `ScheduleEvent`?

This library already has all the interfaces of the AWS resources and could be used to replace `any` in other implementations as well.

Reply:
Yes that's a good idea 👍 

I think something like `import type { ... } from 'aws-lambda'` works

Reply:
@Chriscbr `ScheduleOnTickHandlerClient` implements `IScheduleOnTickHandlerClient`, and the interface has `handle(message: string): Promise<void>;`

I can't change the event type without changing the interface that will be use for all future Schedule implementations

Reply:
I just took another look at the spec, the current API we're looking for is:
```js
on_tick(fn: inflight () => void, opts: ScheduleOnTickProps?): void;
```

So actually we don't need to pass anything to the handler - I think this line should just be
```ts
await this.handler.handle();
```
and IScheduleOnTickHandlerClient should be updated accordingly. Let me know if that works

Reply:
The `handler` object came from `IFunctionHandlerClient`
It needs an `event: string` as input, actually I believe it should receive an `event: any` or an `event: { [key:string]: any } ` because each resource sends an event with different parameters

```ts
export interface IFunctionHandlerClient {
  /**
   * Entrypoint function that will be called when the cloud function is invoked.
   * @inflight
   */
  handle(event: string): Promise<void>;
}
```

Reply:
@marciocadev Thanks for explaining. For now let's update IScheduleOnTickHandlerClient so `handle` accepts zero arguments, and in the implementation (ScheduleOnTickHandlerClient) you can simply invoke `this.handler.handle` with an empty string. You're right, we should allow `IFunctionHandlerClient` to support more types like JSON etc. but I think we should leave that change out of this PR since affects a larger surface area of the SDK